content/tutorials/mushr_unity_sim.md

```diff
@@ -0,0 +1,109 @@
+<br>
+
+### Introduction
+The default state prediction system [racecar_state.py](https://github.com/prl-mushr/mushr_base/blob/master/mushr_base/src/racecar_state.py) works well for low-speed situations where the wheel-slippage and body roll aren't significant. However, once you get to higher speeds (3-4 m/s and above), you want to be able to simulate those things as well so that you know your system is robust to slippage, body roll and so on. For this purpose, we use a unity-based simulator to simulate the state of the car with higher physics fidelity. The system provided by the [mushr_unity_sim](https://github.com/naughtyStark/mushr_unity_sim.git) is a drop in replacement for the default state predictor.
+
+<br>
+{{< figure src="/tutorials/mushr_unity_sim/thumbnail.png" width="800" >}} <br>                           
+<br>
+
+### Goal
+The goal of this tutorial is to get the user acquainted to the unity-based simulation system.
+
+
+### Requirements
+Completed the [quickstart](https://mushr.io/tutorials/quickstart/) tutorial.
+Make sure you’re using the sandbox map. The quickstart tutorial explains how to change the map.
+
+
+## Installation:
+Below are the instructions for installing the unity simulation system.
+
+#### Cloning the repository:
+In a terminal, enter the following commands:
+
+{{<highlight bash>}}
+$ cd ~/catkin_ws/src
+$ git clone https://github.com/naughtyStark/mushr_unity_sim.git
+$ cd ~/catkin_ws
+$ catkin_make
+{{</highlight>}}
+
+The last catkin_make command is necessary to make the system recognize mushr_unity_sim as a valid ros package.
+
+Python dependencies: 
+
+{{<highlight bash>}}
+$ pip install -r requirements.txt
+{{</highlight>}}
+
+
+### Running the base example:
+In a new terminal, enter:
+
+{{<highlight bash>}}
+$ cd ~/catkin_ws/src/mushr_unity_sim/unity
+$ ./donkey_sim.x86_64 -batchmode
+{{</highlight>}}
+
+If you have nvidia driver support on your linux machine (god bless), you can run it without the "-batchmode" tag. The tag makes the sim run in the headless mode which allows for higher fps if you don't have said driver support. Since the simulation surroundings/environment have been simplified to reduce the size of the simulator, there isn't much to see except the car itself and other cars if they are spawned.
+
+
+In a new tab:
+
+{{<highlight bash>}}
+$ roslaunch mushr_unity_sim unity_multi.launch
+{{</highlight>}}
+In another new tab:
+{{<highlight bash>}}
+$ rviz -d ~/catkin_ws/src/mushr_unity_sim/rviz/mushr_unity_sim.rviz
+{{</highlight>}}
+
+You should see 4 cars by default. The poses of these cars are set by the pose_init.py file. The car's initial pose is set the same way as done for the default mushr_sim; by publishing a message on the topic: `/car_name/initialpose` of type: [geometry_msgs/PoseWithCovarianceStamped](http://docs.ros.org/en/melodic/api/geometry_msgs/html/msg/PoseWithCovarianceStamped.html) where `car_name` corresponds to the name of the car. 
```

Comment:
mention driving around and what they should see differently

Reply:
the car slips a lot is that to be expected?

Reply:
actually, it's good that you brought that up. I modeled the car's behavior based on how it drove on the surface of my roof, which is smoother than tarmac. I'm not sure which surface you guys drive on and I've not driven on that surface. Could someone from the lab show me what the car looks like when its turning at the limit of it's grip?